cmd/katapult/client_test.go

```diff
@@ -0,0 +1,61 @@
+package main
+
+import (
+	"testing"
+
+	"github.com/krystal/go-katapult"
+	"github.com/krystal/katapult-cli/config"
+	"github.com/stretchr/testify/assert"
+)
+
+func Test_newClient(t *testing.T) {
+	tests := []struct {
+		name string
+
+		apiKey string
+		apiURL string
+		err    string
```

Comment:
Nitpick: I personally tend to call this field `wantErr` just to make it obvious it's the expected error.